docs/userguides/accounts.md

```diff
@@ -134,6 +137,25 @@ ape accounts generate <ALIAS> --word-count <WORDCOUNT>
 
 If you do not use the `--word-count` option, Ape will use the default word count of 12.
 You can use all of these together or separately to control the way Ape creates and displays your account information.
+
+This same functionality is also scriptable with the same inputs as the `generate` command:
+
+```python
+from ape_acounts import generate_account
+
+alias = "my-account"
+passphrase = "mySecurePassphrase"
+hd_path = "m/44'/60'/0'/0/0"
+word_count = 12
```

Comment:
The first example I would just skip the kwargs, to demonstrate they are not necessary. Or just comment that you only need the first two args

Reply:
Yeah, I was torn on this.  Fewer args is more attractive but wanted to demonstrate the available kwargs.  Will this stuff get autodocced?  Can I link to the actual API?

Reply:
You can link to the method from the autodoc, the user guide can just have a simple example and then be like "for more info on using this method, see the method's documentation" with that part as a link..

search around for markdown links that start with `../methoddocs/` to see how it is done

Reply:
ah, I see how the API is documented now. Though none of the core plugins are documented by autodoc currently.  Should I add one for ape_accounts?

Reply:
I added an autodoc page for ape_accounts.